dist/tools/coccinelle/README.md

```diff
@@ -0,0 +1,60 @@
+## Overview
+
+Coccinelle allows testing C code against semantic patches, automatically
+creating diffs on the way. See http://coccinelle.lip6.fr/ for more information.
+
+## Coccinelle in RIOT
+
+RIOT uses Coccinelle as part of the CI static-tests as follows:
+
+- as with cppcheck, every added or modified file will be checked
```

Comment:
I just removed the cppcheck half sentence. Even a link wouldn't explain much, as cppcheck has no documentation on the matter at all.